### PR TITLE
Feature/css styling

### PR DESCRIPTION
**Pull request by pcmueller**
### Is this a fix or a feature?
- Feature.

### What is the change or fix?
- Installed `node-sass` as dependency and converted CSS files to Sass so order to clean up and implement more reusable code.
- I wanted to make it easier for us to be able to start playing around with the UI design, so I created a `_variables.scss` partial file and added a ton of different colors, fonts, effects, etc, to give us more preset options to try out.  The current display is just me having fun - feel free to change it up!

### How should this be tested?
- First you'll need to run `npm i` again to install the `node-sass` package.  
- Run in browser.  It took me a minute to get the figure out the right Sass dependency, so let me know if you're unable to get past initial Sass errors on page load after installation.  
- For me I just had to restart the local server in console a couple times afterwards.

### CODE REVIEW

File: src/_variables.scss [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+        0 0 10px $beau-blue,
+        0 0 15px $beau-blue,
+        0 0 20px $blue-jeans,
+        0 0 35px $blue-jeans,
+        0 0 40px $blue-jeans,
+        0 0 50px $rajah,
+        0 0 75px $rajah;
+    }
+  }
+}
```
Comment (bhohnco) [inline src/_variables.scss:93]: Good call adding this. You'll have to fill us in on some of these stylings!

File: src/components/App/App.js [hunk 2]
```diff
@@ -17,13 +17,13 @@ class App extends Component {
 
   render() {
     return (
-      <div className="App">
+      <div className="App" style={{ backgroundImage: `url(${Background})`}}>
```
Comment (bhohnco) [inline src/components/App/App.js:20]: Keeping it going, nice!

**Review by bhohnco — APPROVED** [review]

File: src/components/App/App.js [hunk 3]
```diff
@@ -1,9 +1,9 @@
 import React, { Component } from 'react';
+import Background from '../../assets/images/black-cloth.jpeg';
```
Comment (battan40) [inline src/components/App/App.js:2]: I like this template, for us going forward.

File: src/components/App/_app.scss [hunk 4]
```diff
@@ -5,21 +5,20 @@
...
+  justify-content: space-evenly;
 }
 
 .nation-display {
   width: 95%;
   height: 100%;
   display: flex;
   justify-content: space-evenly;
-  padding: 5px;
+  margin-top: 1vw;
```
Comment (battan40) [inline src/components/App/_app.scss:23]: Smart switch to view width, here.

File: src/components/Header/_header.scss [hunk 5]
```diff
@@ -2,17 +2,18 @@ header {
   display: grid;
   grid-template-rows: 70% 30%;
   align-items: center;
-  background-color: salmon;
-  height: 12vw;
-  border-bottom: 6px solid black;
+  background-color: $rich-black;
```
Comment (battan40) [inline src/components/Header/_header.scss:5]: yes to sass

File: src/components/Header/Header.js [hunk 6]
```diff
@@ -1,11 +1,11 @@
 import React from 'react';
-import './Header.css';
+import Banner from '../../assets/images/night-sky.jpeg';
 
 const Header = () => {
 
   return (
-    <header>
-      <h1>~Reverb~</h1>
+    <header style={{ backgroundImage: `url(${Banner})`}}>
```
Comment (battan40) [inline src/components/Header/Header.js:7]: I love the name, and maybe we can revisit the name later together.

**Review by battan40 — APPROVED** [review]

**Pull request merged**